Toivoa_Backend/Controller/userController.js

```diff
@@ -1,142 +1,118 @@
-const Users = require('../Models/userModel');
```

Comment:
This won't work. We need the user model in its current state. Ensure you keep the login/create account routes, while also using the code from here:
https://github.com/Minimal-Alexi/ProjectWB/blob/sprint-3/Toivoa_Backend/Controller/userController.js